localboot/grub.go

```diff
@@ -3,43 +3,54 @@ package main
 import (
 	"io/ioutil"
 	"log"
+	"os"
 	"path"
+	"path/filepath"
 	"strings"
 
 	"github.com/systemboot/systemboot/pkg/bootconfig"
 	"github.com/systemboot/systemboot/pkg/crypto"
 )
 
-// List of paths where to look for grub config files. Grub2Paths will look for
-// files with grub2-compatible syntax, GrubLegacyPaths similarly will treat
-// these as grub-legacy config files.
+// List of directories where to look for grub config files. The root dorectory
+// of each mountpoint, these folders inside the mountpoint and all subfolders
```

Comment:
Just rephrase to "List of directories where to recursively look for grub config files"

Reply:
done

Reply:
done